Improvements UI art update

JoesiahYoungLiu opened the pull request:
![Uploading Screenshot 2025-07-03 164746.png…]()
Updated homepage

Not sure what to do with background tho

JoesiahYoungLiu committed:
Updated homepage with pixel art components

JoesiahYoungLiu committed:
uploaded pixel art components

JoesiahYoungLiu committed:
small update to join page ui

ZeLink3123 commented:
![Screenshot 2025-07-03 171418](https://github.com/user-attachments/assets/92626ef4-d123-4881-932f-34c24b403969)
New join page

JoesiahYoungLiu committed:
i have no idea what this is

JoesiahYoungLiu committed:
Updated gitignore

JoesiahYoungLiu committed:
installed packages to resolve errors

JoesiahYoungLiu committed:
Updated UI and design of home page
added new background images for the homepage
added a panel for the logo and buttons container

ZeLink3123 commented:
![Screenshot 2025-07-06 173813](https://github.com/user-attachments/assets/90d9b6f1-6b1b-48b1-90be-f6de063f5458)
Updated homepage

JoesiahYoungLiu committed:
CI/CD fixes

JoesiahYoungLiu committed:
Fixed Mobile view
slight improvement to UI

ZeLink3123 commented:
![Screenshot 2025-07-06 182659](https://github.com/user-attachments/assets/f4ece76f-ebdc-4f9e-96e2-d19624e30620)
New look

JoesiahYoungLiu committed:
Updated UI for joinpage

JoesiahYoungLiu committed:
Fixed global display flex
Partial implementaion of waiting room ui

JoesiahYoungLiu committed:
Completed UI for waiting room header
Completed UI for parrticipants grid, frontend of adding participants untested and will require help with

JoesiahYoungLiu committed:
invalidcodewarning fix

JoesiahYoungLiu committed:
Added Start game button

JoesiahYoungLiu committed:
Fix layout for Joinpage

JoesiahYoungLiu committed:
Added start game button
minor UI fixes
